Indexing/CodeTree.hpp

```diff
@@ -109,7 +109,7 @@ class CodeTree
    */
   struct ILStruct
   {
-    ILStruct(Literal* lit, unsigned varCnt, Stack<unsigned>& gvnStack);
+    ILStruct(const Literal* lit, unsigned varCnt, Stack<unsigned>& gvnStack);
```

Comment:
I don't object to this at all. But aren't (almost) all term/literal pointers morally `const`? What caused you to make this change? Some compiler lint? Just curious.

Reply:
There was a change that enforced this. I'm not sure the change is in the branch anymore, but I thought I'd leave it here if it compiles.